WearTilesKotlin/app/build.gradle

```diff
@@ -46,14 +46,13 @@ android {
 }
 
 dependencies {
-    implementation "org.jetbrains.kotlin:kotlin-stdlib:$kotlin_version"
```

Comment:
The standard library is depended on by default since 1.4, so removed the explicit dependency here.